pytket/tests/circuit_test.py

```diff
@@ -1257,6 +1260,98 @@ def test_phase_order() -> None:
         assert c == c1
 
 
+def test_dummy_box() -> None:
+    resource_data = ResourceData(
+        op_type_count={OpType.T: ResourceBounds(10, 20)},
+        gate_depth=ResourceBounds(5, 8),
+        op_type_depth={OpType.CZ: ResourceBounds(3, 6), OpType.T: ResourceBounds(4, 6)},
+        two_qubit_gate_depth=ResourceBounds(4, 5),
+    )
+    dbox = DummyBox(n_qubits=3, n_bits=1, resource_data=resource_data)
+    c = Circuit(4, 2)
+    c.add_dummybox(dbox, [0, 2, 3], [1])
+    cmds = c.get_commands()
+    assert len(cmds) == 1
+    op = cmds[0].op
+    assert type(op) is DummyBox
+    resource_data1 = op.get_resource_data()
+    op_type_count = resource_data1.get_op_type_count()
+    assert op_type_count[OpType.T].get_min() == 10
+    assert op_type_count[OpType.T].get_max() == 20
+
```

Comment:
Could you also call `json_validate` on `c`?

Reply:
Done.